brave/src/main/java/com/linecorp/armeria/common/brave/RequestContextCurrentTraceContext.java

```diff
@@ -80,7 +84,8 @@ public static void setCurrentThreadNotRequestThread(boolean value) {
         }
     }
 
-    private static final CurrentTraceContext DEFAULT = new RequestContextCurrentTraceContext(new Builder());
+    private static final RequestContextCurrentTraceContext DEFAULT =
+            new RequestContextCurrentTraceContext(new Builder());
```

Comment:
Could we add some sensible default patterns here, such as `^RMI TCP Connection`?

Reply:
I'm not sure what are sensible defaults - I noticed `RMI TCP Connection` from the posted stacktrace but can't find any documentation on it, it's probably a JDK implementation detail. So maybe it's best to just leave them to the users.